processor/attributesprocessor/factory.go

```diff
@@ -129,7 +130,10 @@ func buildAttributesConfiguration(config Config) ([]attributeAction, error) {
 			} else {
 				action.FromAttribute = a.FromAttribute
 			}
-
+		case HASH:
+			if a.Value != nil || a.FromAttribute != "" {
```

Comment:
Could you please extend this behavior to the delete action? This one is better, delete just ignores value and from_attribute...